db/migrate/20160824050520_create_reviews.rb

```diff
@@ -0,0 +1,14 @@
+class CreateReviews < ActiveRecord::Migration[5.0]
+  def change
+    create_table :reviews do |t|
+      t.integer :school_id, null: false
+      # review_type is to represent the different types of reviews, eg: "My kid went here", "I heard about it..", etc
+      t.string :review_type, null: false
+      t.integer :score
```

Comment:
Yet to be determined beyond "Thumbs Up", "Thumbs Down" and Neutral.  Current thinking is potentially 1 (down), 2 (neutral), 3 (up)?  Happy to change if there is a better option